docs/MO_DG/prepare_model/Getting_performance_numbers.md

```diff
@@ -1,28 +1,25 @@
 # Getting Performance Numbers {#openvino_docs_MO_DG_Getting_Performance_Numbers}
 
-
-## Tip 1. Measure the Proper Set of Operations 
-
-When evaluating performance of your model with the OpenVINO Runtime, you must measure the proper set of operations. To do so, consider the following tips: 
+When evaluating performance of model with the OpenVINO Runtime, it is required to measure the proper set of operations. To do so, consider the tips below: 
 
 - Avoid including one-time costs like model loading.
 
-- Track separately the operations that happen outside the OpenVINO Runtime, like video decoding. 
+- Track separately operations which occurred outside of the OpenVINO Runtime (like video decoding). 
 
-> **NOTE**: Some image pre-processing can be baked into the IR and accelerated accordingly. For more information, refer to [Embedding the Preprocessing](Additional_Optimizations.md). Also consider [Runtime Optimizations of the Preprocessing](../../optimization_guide/dldt_deployment_optimization_common).
+> **NOTE**: Some image pre-processing can be baked into the IR and accelerated accordingly. For more information, refer to the [Embedding the Preprocessing guide](Additional_Optimizations.md). Additional information are also present in the [Runtime Optimizations of the Preprocessing guide](../../optimization_guide/dldt_deployment_optimization_common).
```

Comment:
```suggestion
> **NOTE**: Some image pre-processing can be baked into the IR and accelerated accordingly. For more information, refer to the [Embedding the Preprocessing ](Additional_Optimizations.md) guide. Additional information is also present in the [Runtime Optimizations of the Preprocessing](../../optimization_guide/dldt_deployment_optimization_common) guide.
```
IMHO it is better to leave out the guide/page/article words from linking syntax, as it highlights the name/topic of a particular document. Additional words in titles might result in some confusion.